src/Microsoft.VisualStudio.Editors/MyApplication/MyApplicationCodeGenerator.vb

```diff
@@ -361,6 +393,25 @@ Namespace Microsoft.VisualStudio.Editors.MyApplication
             Method.Statements.Add(Statement)
         End Sub
 
+        ' Overload of AddFieldAssignment to accept a String instead of a Type for EnumType '
+        ' Background: For the case of HighDpiMode, the actual Type is only available in the Designer OOP server-side.
+        ' So to use types, we would need to generate this also in the server-side.
+        ' We don 't think it's necessary though, and consider it is too much of an effort,
+        ' so we decided to use a String for the type name client side.
+        Private Shared Sub AddFieldAssignment(Method As CodeMemberMethod, FieldName As String, EnumType As String, EnumFieldName As String)
+            Dim Statement As CodeAssignStatement
+            Statement = New CodeAssignStatement With {
```

Comment:
So, just for my own education:
How do you use capitalization of variable names in this (VB) repo. In the VB AppFramework, we still use small caps for parameters and local variables, and then also pretty much the same style as in C# for consts and member variables/backing fields.

Reply:
Seems like the code convention for our VB files is a mix of pascal case and camel case for local variables, camel case for parameters, and again a mix of both plus all-caps for global variables. I might open another PR to address this, and try to have a more cohesive convention; it's not that frequent that we add/edit these files, but it would help in readability and code maintenance.